### DISCUSSION TITLE
Loss dont add up

### DISCUSSION THREAD

**Discussion by jaideep11061982** [rwightman/efficientdet-pytorch#230] [General]
Any reason why Class loss+boxloss not equal to total loss

`loss:  0.58021,box_loss: 0.00468 , cls_loss: 0.34635`

**Comment by mikeb127** [comment]
Due to the box loss weight. See line 219 of the loss.py file. The box loss weight is taken from the model config files

 _total_loss = cls_loss + box_loss_weight * box_loss_
